error_tracker/django/migrations/0002_auto_20201018_1311.py

```diff
@@ -1,23 +0,0 @@
-# Generated by Django 3.1.2 on 2020-10-18 11:11
-
```

Comment:
We would need this file.

Reply:
As you ask previously if we really need it... Will be restored on next commit.

Reply:
We need this file but app name is not correct, we can connect over zoom/goole hangout to see what's not working you but same is working for me.